src/sentry/utils/cache.py

```diff
@@ -9,11 +9,17 @@
 
 import functools
 
-from django.core.cache import cache
+from django.core.cache import cache, get_cache, InvalidCacheBackendError
 
 default_cache = cache
 
 
+try:
```

Comment:
This makes more sense in `sentry.preprocess_hashes`, IMO, which might itself be better in `sentry.filters.discarded_hashes` or something?

Reply:
👍  will do that

Reply:
Yeah, agree with this.